portal/common/lib/routing.test.ts

```diff
@@ -2,41 +2,51 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { getRoutes, matchPathToRoute } from "./routing";
-import { test, expect } from 'vitest';
+import { test, expect } from "vitest";
 import { SuiClient, getFullnodeUrl } from "@mysten/sui/client";
 import { NETWORK } from "./constants";
 
-const snakeSiteObjectId = '0x3e01b1b8bf0e54f7843596345faff146f1047e304410ed2eb85d5f67ad404206';
-test.skip('getRoutes', async () => {
+const snakeSiteObjectId = "0x3e01b1b8bf0e54f7843596345faff146f1047e304410ed2eb85d5f67ad404206";
```

Comment:
Nit: Since the contract was updated, the snake site object Id should also be updated. 

Even though we skip this test we still use it occasionally for local development.